docs/zh/deployment/quick-start.md

```diff
@@ -66,60 +66,26 @@ Quick Start只针对本地测试使用，所以一般用户不需要自己下载
 2. 在根目录下执行`mvn clean package -pl apollo-assembly -am -DskipTests=true`
 3. 复制apollo-assembly/target下的jar包，rename为apollo-all-in-one.jar
 
-# 二、安装步骤
-## 2.1 创建数据库
-Apollo服务端共需要两个数据库：`ApolloPortalDB`和`ApolloConfigDB`，我们把数据库、表的创建和样例数据都分别准备了sql文件，只需要导入数据库即可。
+# 二、数据库初始化及配置
 
-> 注意：如果你本地已经创建过Apollo数据库，请注意备份数据。我们准备的sql文件会清空Apollo相关的表。
+本部分以h2数据库为例，介绍如何初始化数据库及配置数据库连接信息。由于h2数据库是内存数据库，数据的初始化和访问都是在内存中进行的，所以不需要安装和导入。并且，由于h2数据库是内存数据库，所以重启机器后，数据会丢失，不推荐在生产环境使用。 如果需要使用其他数据库，如MySQL，可以参考[使用其他数据库](#五使用其它数据库)
```

Comment:
Need add version description?

Reply:
@Shoothzj apollo's version. 

If those feature start from a version x.y.z, we need tell user, 

if all version of apollo have those feature, it's ok to hide apollo's version.

Reply:
It might be beneficial to include a note informing users that support for h2 necessitates version 2.2.0 or higher.I think we may add a note to tell the users that h2 support requires 2.2.0+ version.